tox/config.py

```diff
@@ -1171,31 +1172,34 @@ def getargvlist(cls, reader, value):
         return commands
 
     @classmethod
-    def processcommand(cls, reader, command):
+    def processcommand(cls, reader, command, replace=True):
         posargs = getattr(reader, "posargs", "")
         posargs_string = list2cmdline([x for x in posargs if x])
 
         # Iterate through each word of the command substituting as
         # appropriate to construct the new command string. This
         # string is then broken up into exec argv components using
         # shlex.
-        newcommand = ""
-        for word in CommandParser(command).words():
-            if word == "{posargs}" or word == "[]":
-                newcommand += posargs_string
-                continue
-            elif word.startswith("{posargs:") and word.endswith("}"):
-                if posargs:
+        if replace:
+            newcommand = ""
+            for word in CommandParser(command).words():
+                if word == "{posargs}" or word == "[]":
                     newcommand += posargs_string
                     continue
-                else:
-                    word = word[9:-1]
-            new_arg = ""
-            new_word = reader._replace(word)
-            new_word = reader._replace(new_word)
-            new_word = new_word.replace('\\{', '{').replace('\\}', '}')
-            new_arg += new_word
-            newcommand += new_arg
+                elif word.startswith("{posargs:") and word.endswith("}"):
+                    if posargs:
+                        newcommand += posargs_string
+                        continue
+                    else:
+                        word = word[9:-1]
+                new_arg = ""
+                new_word = reader._replace(word)
+                new_word = reader._replace(new_word)
+                new_word = new_word.replace('\\{', '{').replace('\\}', '}')
+                new_arg += new_word
+                newcommand += new_arg
+        else:
+            newcommand = command
```

Comment:
**Note to reviewers**: the new `else` clause here is essential to the change.  When `replace=False`, we should leave the command uninterpreted (we can't evaluate it since we're potentially missing some of the variables).